docs/IntelOwl/advanced_configuration.md

```diff
@@ -4,34 +4,33 @@ This page includes details about some advanced features that Intel Owl provides
 
 ## ElasticSearch
 
-Right now only ElasticSearch v8 is supported.
-
-### DSL
+_Available for version > 6.1.0_
 
-IntelOwl makes use of [django-elasticsearch-dsl](https://django-elasticsearch-dsl.readthedocs.io/en/latest/about.html) to index Job results into elasticsearch. The `save` and `delete` operations are auto-synced so you always have the latest data in ES.
+Right now only ElasticSearch v8 is supported.
 
+### Configuration
 In the `env_file_app_template`, you'd see various elasticsearch related environment variables. The user should spin their own Elastic Search instance and configure these variables.
 
-#### Kibana
-
-Intel Owl provides a Kibana's "Saved Object" configuration (with example dashboard and visualizations). It can be downloaded from [here](https://github.com/intelowlproject/IntelOwl/blob/develop/configuration/Kibana-Saved-Conf.ndjson) and can be imported into Kibana by going to the "Saved Objects" panel (http://localhost:5601/app/management/kibana/objects).
+* ELASTIC_HOST: URL of the Elasticsearch instance.
+* ELASTIC_PASSWORD: (optional) Password of the "elastic" user. This can be empty in case of external services with credentials in the url.
+* ELASTICSEARCH_BI_ENABLED: Use the Business Intelligence feature.
```

Comment:
There is no explanation regarding what the Business Intelligence feature is I guess. Can you add a reference to that section?